src/modules/Dashboard/features/Applet/AddUser/LinkGenerator/LinkGenerator.tsx

```diff
@@ -8,6 +8,7 @@ import { useAsync } from 'shared/hooks/useAsync';
 import { StyledFlexTopCenter, StyledTitleBoldMedium, theme } from 'shared/styles';
 import { Tooltip } from 'shared/components';
 
+import { Mixpanel } from 'shared/utils';
```

Comment:
Please fix the import styles issues everywhere you made changes. 
As I understand, "shared" imports should be grouped together. And every group should be separated from each other by empty lines.
@IhorNesterenko99 is that correct?

Reply:
yes)